.github/workflows/plugin-cd.yml

```diff
@@ -41,16 +41,28 @@ jobs:
       - name: ci/build
         uses: mattermost/actions/plugin-ci/build@0d1a5aa0352d9030d51dd6f01868351cad80ef0a
 
-  release:
+  release-github:
```

Comment:
I will suggest first have the first of `release-s3` and `release-github` depends on S3. Mostly just to ensure that we won't end up with a github release only.